homeworks/rostyslav.vyshemirskyi_Rooosss/PokemonProject/index.html

```diff
@@ -13,9 +13,27 @@
         <title>PokemonProject</title>
     </head>
     <body>
+        <header class="header">
+            <div class="container header__container">
+                <div class="header__inner">
+                    <img src="./image/logo.svg" alt="Logo">
+                    <nav class="navigation">
+                        <ul class="navigation__list">
+                            <li class="navigation__item">
+                                <a class="navigation__link" href="pokemons.html?_ijt=7j4qnubtedq9o7imka7390jf82&_ij_reload=RELOAD_ON_SAVE">Pokemons</a>
+                            </li>
+                            <li class="navigation__item">
+                                <a class="navigation__link" href="#">About</a>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+            </div>
+        </header>
         <main class="main">
-            <section class="about__section">
-                <div class="about_container">
+            <div class="container">
+                <section class="about__section">
+                    <img src="../PokemonProject/image/pokemon-about.svg" width="532" height="297" alt="Pokemon-about">
```

Comment:
Hey, @Rooosss, so in the path you are going up (`../`) and going back (`PokemonProject/`). You can skip it as you do fo logo image